### PR TITLE
2024 santa cruz fp

### PR DESCRIPTION
**Pull request by thomasgtsr**

### CODE REVIEW

**Comment by thomasgtsr** [comment]
Sorry I should have stated this in the description, but the goal is to add the fingerprints for the 2024 Hyundai Santa Cruz

**Comment by sshane** [comment]
| Dongle, platform, VIN                                                                                                                 | Name from VIN 🆚 CarInfo                                                                            | Segments                | Bad seg tags   | Good seg tags                                                                       |
|:--------------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------------------------------------------------------------------|:------------------------|:---------------|:------------------------------------------------------------------------------------|
| [6d50867a463d35f6](https://useradmin.comma.ai/?onebox=6d50867a463d35f6)<br><sub>HYUNDAI SANTA CRUZ 1ST GEN<br>5NTJEDAF5........</sub> | HYUNDAI Santa Cruz 2024 🆚<br>Hyundai Santa Cruz 2022-23<br><sub>🎉 <b>New model year!</b> 🎉</sub> | 18 routes,<br>3.1 hours |                | - valid torque params: 114<br>- all lat active: 41<br>- no input all lat active: 28 |

**Comment by sshane** [comment]
Looks like a false positive too high torque failure, though it is rather high. I checked other Santa Cruz's we support and they all are around 3.5 m/s^2 so this isn't a divergence (not sure why we put 2.7 m/s^2 when porting the car originall)

image

**Comment by thomasgtsr** [comment]
Understood. Does that mean the 2024 car will be supported? Sorry I'm not good at reading the results.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

File: selfdrive/car/hyundai/fingerprints.py [hunk 1]
```diff
@@ -1595,6 +1595,7 @@
     (Ecu.fwdCamera, 0x7c4, None): [
       b'\xf1\x00NX4 FR_CMR AT USA LHD 1.00 1.00 99211-CW000 14M',
       b'\xf1\x00NX4 FR_CMR AT USA LHD 1.00 1.00 99211-CW010 14X',
+      b'\xf1\x00NX4 FR_CMR AT USA LHD 1.00 1.00 99211-CW020 14Z',
```
Comment (sshane) [inline selfdrive/car/hyundai/fingerprints.py:1598]: wonder what comes next

**Comment by thomasgtsr** [comment]
Lol. Something outside the alphabet.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Pull request merged**